cpp/src/link_prediction/similarity_impl.cuh

```diff
@@ -36,13 +36,16 @@
 namespace cugraph {
 namespace detail {
 
+enum Coefficient { JACCARD, SORENSEN, OVERLAP, COSINE };
+
 template <typename vertex_t, typename edge_t, typename weight_t, bool multi_gpu, typename functor_t>
 rmm::device_uvector<weight_t> similarity(
   raft::handle_t const& handle,
   graph_view_t<vertex_t, edge_t, false, multi_gpu> const& graph_view,
   std::optional<edge_property_view_t<edge_t, weight_t const*>> edge_weight_view,
   std::tuple<raft::device_span<vertex_t const>, raft::device_span<vertex_t const>> vertex_pairs,
   functor_t functor,
+  Coefficient coff,
```

Comment:
I google searched for "coefficient abbreviation" and got coeff or coef.

coff returned "common object file format".

coeff might be a better abbreviation.

Reply:
Changed.